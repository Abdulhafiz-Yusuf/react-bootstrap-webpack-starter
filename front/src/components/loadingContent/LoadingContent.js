// @flow

// #region imports
import React from 'react';
// #endregion

// #region flow types
type Props = {
  ...any,
};
// #endregion

function LoadingContent(props: Props) {
  return <div>loading...</div>;
}

export default LoadingContent;
