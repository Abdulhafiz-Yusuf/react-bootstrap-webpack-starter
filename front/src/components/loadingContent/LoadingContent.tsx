import React from 'react';

type Props = {};

function LoadingContent(props: Props) {
  return <div>loading...</div>;
}

LoadingContent.displayName = 'LoadingContent';

export default LoadingContent;
