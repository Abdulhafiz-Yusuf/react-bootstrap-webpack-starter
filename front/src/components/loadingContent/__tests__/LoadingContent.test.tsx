import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/react/cleanup-after-each';
import LoadingContent from '../LoadingContent';

let rootElement: any = null;

describe('LoadingContent component', () => {
  beforeEach(() => {
    rootElement = document.createElement('div');
    document.body.appendChild(rootElement);
  });

  afterEach(() => {
    rootElement && document.body.removeChild(rootElement);
    rootElement = null;
  });

  it('renders as expected', () => {
    const { container } = render(<LoadingContent />, rootElement);
    expect(container.firstChild).toMatchSnapshot();
  });
});
