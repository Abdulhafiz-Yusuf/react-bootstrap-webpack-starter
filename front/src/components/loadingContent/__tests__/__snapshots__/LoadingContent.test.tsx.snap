// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingContent component renders as expected 1`] = `
<div>
  loading...
</div>
`;
